worlds/tunic/options.py

```diff
@@ -1,28 +1,36 @@
 from dataclasses import dataclass
 
-from Options import DefaultOnToggle, Toggle, StartInventoryPool, Choice, Range, TextChoice, PerGameCommonOptions
+from Options import (DefaultOnToggle, Toggle, StartInventoryPool, Choice, Range, TextChoice, PerGameCommonOptions,
+                     OptionGroup)
 
 
 class SwordProgression(DefaultOnToggle):
-    """Adds four sword upgrades to the item pool that will progressively grant stronger melee weapons, including two new swords with increased range and attack power."""
+    """
+    Adds four sword upgrades to the item pool that will progressively grant stronger melee weapons, including two new swords with increased range and attack power.
+    """
     internal_name = "sword_progression"
     display_name = "Sword Progression"
 
 
 class StartWithSword(Toggle):
-    """Start with a sword in the player's inventory. Does not count towards Sword Progression."""
+    """
+    Start with a sword in the player's inventory. Does not count towards Sword Progression.
+    """
     internal_name = "start_with_sword"
     display_name = "Start With Sword"
 
 
 class KeysBehindBosses(Toggle):
-    """Places the three hexagon keys behind their respective boss fight in your world."""
+    """
+    Places the three hexagon keys behind their respective boss fight in your world.
+    """
     internal_name = "keys_behind_bosses"
     display_name = "Keys Behind Bosses"
 
 
 class AbilityShuffling(Toggle):
-    """Locks the usage of Prayer, Holy Cross*, and the Icebolt combo until the relevant pages of the manual have been found.
+    """
+    Locks the usage of Prayer, Holy Cross*, and the Icebolt combo until the relevant pages of the manual have been found.
     If playing Hexagon Quest, abilities are instead randomly unlocked after obtaining 25%, 50%, and 75% of the required Hexagon goal amount.
     *Certain Holy Cross usages are still allowed, such as the free bomb codes, the seeking spell, and other player-facing codes.
```

Comment:
If you're adding spaces between asterisks and text, there's another one here.

Reply:
Good catch, thanks!